Formula/llvm.rb

```diff
@@ -89,7 +91,9 @@ def install
       libunwind
     ]
 
-    py_ver = "3.9"
+    python = Formula["python@3.9"]
+    py_ver = Language::Python.major_minor_version(python)
```

Comment:
Not for this PR buit we should deprecate using this at some point. I see minimal benefits now that we depend on specific versions of Python. It's a lot of characters to say `3.9`.

Reply:
Not a fan of this either, I just think it's less easily missed when we have to migrate to `python@3.10`.

DSL that would allow us to easily refer to the Python-formula-that-this-formula-depends-on would be nice tho.

Reply:
That's possible. If we want to avoid passing `self` as a argument, then we would require a new `include` line. `Language::Python::Shebang` is an example of what I did for automatic python detection.